lgsf/polling_stations/commands.py

```diff
@@ -0,0 +1,97 @@
+from rich.progress import Progress
+
+from lgsf.commands.base import PerCouncilCommandBase
+from lgsf.path_utils import load_scraper, load_council_info
+
+
+class Command(PerCouncilCommandBase):
```

Comment:
Feels like a large chunk of this file could be refactored in to a common file - it's largely the same as the councillors one, and it would be good to keep the outputs and interfaces as similar as we can I think.

This can be an issue of course, not needed for this PR